aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -459,3 +471,48 @@ proptest! {
         );
     }
 }
+
+#[test]
+#[should_panic]
+fn test_aggregator_snapshot_not_equivalent_gas() {
+    let test_env = TestEnvConfig {
+        executor_mode: ExecutorMode::BothComparison,
+        aggregator_execution_mode: AggregatorMode::BothComparison,
+        block_split: BlockSplit::Whole,
+    };
+
+    println!("Testing test_aggregator_snapshot {:?}", test_env);
+    let element_type = ElementType::U64;
+    let use_type = UseType::UseResourceType;
+
+    let mut h = setup(
+        test_env.executor_mode,
+        test_env.aggregator_execution_mode,
+        9,
```

Comment:
nit: there are less txns created than 9?

Reply:
it would be nice to not need to write this explicitly

Reply:
indeed